### DISCUSSION TITLE
Is there a user's guide or something to get an old guy started?😂

### DISCUSSION THREAD

**Discussion by Bridger18** [frizLabz-FFriZz/PineScript-v6-vscode#9] [Show and tell]
Hi, I know it is a lot of work, but some guide or manual would be great help, Thanks

**Comment by Fred-Vatin** [comment]
Hi,

Until now, I found it pretty straight forward to use. When you want to use library, create a folder whose name is the same as in tradingView and put your libs there. Then you can import as you would in TV. Autocomplete works.

Do you have specific questions ?

**Reply by JakeTheSnake3p0** [reply]
I followed your instruction and renamed the directory the libraries are in to my TV name (ex., if I would normally "import JTS/statistics as stats", that means the library directory name should be "JTS"). However, I'm unable to preview another library's contents on hover of the import line.

Should the root directory name of my repository be "JTS", or should JTS be a folder within that root?
